sneak_peek.md

```diff
@@ -17,21 +17,25 @@ This is essentially a placeholder for the next release note ...
     fixed-size variables followed by record variables.
 
 * New Limitations
-  + ADIOS driver are ready only. No vard, varn, low-level, and nonblocking support.
+  + ADIOS driver is ready only. There is also no vard, varn and low-level API support.
```

Comment:
Mentioning read-only is sufficient in this file, Direct readers to README.ADIOS or README.BP for details.